lib/actions/upsert.js

```diff
@@ -1,19 +1,17 @@
-'use strict';
-const jsforce = require('jsforce');
 const lookup = require('./lookup');
-const debug = require('debug')('salesforce:upsert');
-const SALESFORCE_API_VERSION = process.env.SALESFORCE_API_VERSION || '45.0';
 const MetaLoader = require('../helpers/metaLoader');
+const attachment = require('../helpers/attachment.js');
+const sfConnection = require('../helpers/sfConnection.js');
 
 /**
  * This function will return a metamodel description for a particular object
  *
  * @param configuration
  */
-module.exports.getMetaModel = async function getMetaModel(configuration) {
+module.exports.getMetaModel = function getMetaModel(configuration) {
```

Comment:
Why it is not an `async` function?

Reply:
Here you are right. For the consistency it's better write `async`